Tests/Debug/TraceableDomainEventDispatcherTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Biig\Component\Domain\Tests\Debug;
+
+use Biig\Component\Domain\Debug\TraceableDomainEventDispatcher;
+use Biig\Component\Domain\Event\DomainEvent;
+use Biig\Component\Domain\Event\DomainEventDispatcher;
+use Biig\Component\Domain\Exception\InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class TraceableDomainEventDispatcherTest extends TestCase
+{
+    public function testItThrowsExceptionIfNotDomainEvent()
```

Comment:
This test the decorated, not the decorator.

Reply:
Actually it does test the decorator, or I miss something.

Reply:
Oh. My bad I wanted to delete this comment and I forgot. You right. :)